vsintegration/src/FSharp.Editor/CommentUncommentService.fs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Microsoft.VisualStudio.FSharp.Editor
+
+open System.ComponentModel.Composition
+open Microsoft.CodeAnalysis.Editor
+open Microsoft.CodeAnalysis.Shared.Extensions
+open Microsoft.CodeAnalysis.Text
+open Microsoft.CodeAnalysis.Text.Shared.Extensions
+open Microsoft.VisualStudio.Text
+open Microsoft.VisualStudio.Text.Operations
+open Microsoft.VisualStudio.Utilities
+open Microsoft.CodeAnalysis.Editor.Implementation.CommentSelection
+open Microsoft.CodeAnalysis.Host.Mef
+open Microsoft.CodeAnalysis
+open System.Composition
```

Comment:
no idea. errors are not highlighted in vs 2015 on this project. Ionide highlight almost everything as errors...